vllm/v1/core/sched/scheduler.py

```diff
@@ -1414,8 +1416,26 @@ def reset_prefix_cache(self, reset_running_requests: bool = False) -> bool:
                 "the presence of running requests waiting for remote KV transfer, "
                 "which is not supported yet."
             )
+
+        if reset_connector:
+            reset_successful = self.reset_connector_cache() and reset_successful
+
         return reset_successful
 
+    def reset_connector_cache(self) -> bool:
+        if self.connector is None:
+            logger.warning("reset_connector called but no KV connector is configured.")
+            return False
+
+        if self.connector.reset_cache() is False:
+            return False
```

Comment:
nit: we could probably add a debug log for calls that require resetting a cache on a connector that does not implement resetting (when None is returned), but this is not really a priority.